backend/swpp/model/tutor.py

```diff
@@ -1,7 +1,12 @@
 from django.db import models
 from annoying.fields import AutoOneToOneField
 from swpp.model.profile import Profile
+from swpp.model.lecture import Lecture
+from swpp.model.times import Times
 
 class Tutor(models.Model):
     profile = AutoOneToOneField(Profile, primary_key=True, on_delete=models.CASCADE)
     bio = models.TextField()
+    exp = models.TextField()
+    lectures = models.ManyToManyField(Lecture)
+    times = AutoOneToOneField(Times, on_delete=models.CASCADE)
```

Comment:
django가 뭐라 하니까
```suggestion
    times = AutoOneToOneField(Times, on_delete=models.CASCADE, null=True)
```
로 고쳐줘요

Reply:
아 글고 OneToOne 쓰면 안 될것 같아요
Tutor 말고 다른거랑도 대응될 수 있으니까 (request 등)
ForeignKey를 쓰시는게 좋겠네요